lib/server/services/dpl/DplDetectorsService.js

```diff
@@ -26,6 +26,20 @@ class DplDetectorsService {
         const detectors = await DplDetectorRepository.findAll();
         return detectors.map(dplDetectorAdapter.toEntity);
     }
+
+    /**
+     * Find DPL detectors which flags contribute to GAQ for given run and data pass
+     *
+     * @param {number} dataPassId id of data pass
+     * @param {number} runNumber run number
+     * @return {Promise<Partial<DplDetector>[]>} promise of DPL detectors
+     */
+    async getGaqDetectors(dataPassId, runNumber) {
+        const detectors = await DplDetectorRepository.findAll({
+            include: [{ association: 'gaqAssociation', where: { dataPassId, runNumber } }],
+        });
+        return detectors.map(dplDetectorAdapter.toMinifiedEntity);
```

Comment:
As said previously this is an error, this should be in something like `gaqService`. Same applies for router, controller

Reply:
moved to DataPassesService